docs/resources/rule_amqp_external

```diff
@@ -0,0 +1,95 @@
+    url = "amqps://test.com"
+    routing_key = "new:key"
+    mandatory_route = true
+    persistent_messages = true
+    message_ttl = 55
+    headers = [
+      {
+        name : "User-Agent",
+        value : "user-agent-string",
+      },
+      {
+        name : "User-Agent-Extra",
+        value : "user-agent-string",
+      },
+    ]
+    signing_key_id = ably_api_key.api_key_1.id
+  }
+}
+```
+
+<!-- schema generated by tfplugindocs -->
+## Schema
+
+### Required
+
+- `app_id` (String) The Ably application ID.
+- `source` (Attributes) object (rule_source) (see [below for nested schema](#nestedatt--source))
+- `target` (Attributes) object (rule_source) (see [below for nested schema](#nestedatt--target))
+
+### Optional
+
+- `request_mode` (String) This is Single Request mode or Batch Request mode. Single Request mode sends each event separately to the endpoint specified by the rule
+- `status` (String) The status of the rule. Rules can be enabled or disabled.
+
+### Read-Only
+
+- `id` (String) The rule ID.
+
+<a id="nestedatt--source"></a>
+### Nested Schema for `source`
+
+Required:
+
+- `channel_filter` (String)
+- `type` (String)
+
+
+<a id="nestedatt--target"></a>
+### Nested Schema for `target`
+
+Required:
+
+- `mandatory_route` (Boolean) Reject delivery of the message if the route does not exist, otherwise fail silently.
+- `persistent_messages` (Boolean) Marks the message as persistent, instructing the broker to write it to disk if it is in a durable queue.
+- `routing_key` (String) The Kafka partition key. This is used to determine which partition a message should be routed to, where a topic has been partitioned. routingKey should be in the format topic:key where topic is the topic to publish to, and key is the value to use as the message key
+- `url` (String) The webhook URL that Ably will POST events to
+
+Optional:
+
+- `enveloped` (Boolean) Delivered messages are wrapped in an Ably envelope by default that contains metadata about the message and its payload. The form of the envelope depends on whether it is part of a Webhook/Function or a Queue/Firehose rule. For everything besides Webhooks, you can ensure you only get the raw payload by unchecking `Enveloped` when setting up the rule
+- `format` (String) JSON provides a simpler text-based encoding, whereas MsgPack provides a more efficient binary encoding
+- `headers` (Attributes List) If you have additional information to send, you'll need to include the relevant headers (see [below for nested schema](#nestedatt--target--headers))
+- `message_ttl` (Number) You can optionally override the default TTL on a queue and specify a TTL in minutes for messages to be persisted. It is unusual to change the default TTL, so if this field is left empty, the default TTL for the queue will be used.
+
+<a id="nestedatt--target--headers"></a>
+### Nested Schema for `target.headers`
+
+Required:
+
+- `name` (String) The name of the header
+- `value` (String) The value of the header
```

Comment:
Blank line required at end of file?

Reply:
Was wondering if there should be a blank line? 
![image](https://user-images.githubusercontent.com/25511700/193231061-2e1eb643-b784-453e-9dc0-274e79472525.png)

Reply:
Ah, I see. So that's how the automatic markdown generator generates them. I can force add line if needed via template file - should I do that?

Reply:
No, I don't think it's a blocker. If that's what gets generated I think no need to manually change it.

Reply:
Great, thanks!